Core/Types/CkanModule.cs

```diff
@@ -552,6 +566,21 @@ public CkanModule(string json, IGameComparator comparator)
                     throw new BadMetadataKraken(null, error);
                 }
             }
+
+            // Calculate the Searchables.
+            CalculateSearchables();
+        }
+
+        /// <summary>
+        /// Calculate the mod properties used for searching via Regex.
+        /// </summary>
+        public void CalculateSearchables()
+        {
+            SearchableIdentifier  = identifier  == null ? string.Empty : CkanModule.nonAlphaNums.Replace(identifier, "");
+            SearchableName        = name        == null ? string.Empty : CkanModule.nonAlphaNums.Replace(name, "");
+            SearchableAbstract    = @abstract   == null ? string.Empty : CkanModule.nonAlphaNums.Replace(@abstract, "");
+            SearchableDescription = description == null ? string.Empty : CkanModule.nonAlphaNums.Replace(description, "");
+            SearchableAuthors     = author      == null ? string.Empty : CkanModule.nonAlphaNums.Replace(String.Join("", author), "");
```

Comment:
I'm not sure concatenating the authors like this is what we want. If my authors list is `[ "abc", "def", "ghi" ]`, then this would match on "bcde", which isn't part of any author's name and depends on the ordering of the list. Can we apply the regex to each author's name individually but keep them as separate list elements?

Reply:
Do you think there's a high chance that someone might encounter it?
Even if, this probably only results in very few false positives.

Reply:
Well if someone wants to search for droric with "dro", this will cause Asteroid Day to match incorrectly:

![image](https://user-images.githubusercontent.com/1559108/56665144-1ae30780-666f-11e9-9d2f-2f4b3c39e8f0.png)

But in general why not do it right since we know about it?

Reply:
Okay, changing it :+1: